src/packages/@workspaces/features/collection-list/layout/CollectionList.svelte

```diff
@@ -186,183 +167,102 @@
+      class="d-flex align-items-center p-2 rounded-1"
+      style="cursor:pointer; justify-content: space-between; height:32px;
+      background-color: {isHovered
+        ? 'var(--dropdown-option-hover)'
+        : 'transparent'};"
+      on:mouseover={handleMouseOver}
+      on:mouseout={handleMouseOut}
+      on:click={toggleExpandCollection}
     >
-      <p class="mb-0 text-whiteColor ellipsis text-fs-16">
-        {$currentWorkspace?.name || ""}
-      </p>
-      <button
-        class=" border-0 rounded px-2 angleButton"
-        on:click={() => {
-          leftPanelController.leftPanelCollapse =
-            !leftPanelController.leftPanelCollapse;
-          leftPanelController.handleCollapseCollectionList();
-        }}
-        id="doubleAngleButton"
+      <div class="d-flex align-items-center ps-2">
+        <img
+          src={angleRight}
+          class="me-3"
+          style="height:8px; width:4px; margin-right:8px; {isExpandCollection
+            ? 'transform:rotate(90deg);'
+            : 'transform:rotate(0deg);'}"
+          alt="angleRight"
+        />
+
+        <CollectionIcon
+          height={"12px"}
+          width={"12px"}
+          color={"var(--icon-secondary-130)"}
+        />
+
+        <p class="ms-2 mb-0 sparrow-fs-13" style="font-weight: 500;">
+          Collections
+        </p>
+      </div>
+
+      <Tooltip
+        title={"Add Collection"}
+        placement={"bottom"}
+        distance={13}
+        show={isHovered}
+        zIndex={701}
       >
-        <img src={doubleangleLeft} alt="" class="filter-green" />
-      </button>
-    </div>
-    <div
-      class="d-flex align-items-center justify-content-between px-2 pt-3 gap-1"
-    >
-      <Input
-        id="collection-list-search"
-        width={"100%"}
-        height={"33px"}
-        type="search"
-        bind:value={searchData}
-        on:input={(e) => {
-          handleSearch();
-        }}
-        defaultBorderColor="transparent"
-        hoveredBorderColor="var(--border-primary-300)"
-        focusedBorderColor={"var(--border-primary-300)"}
-        class="text-fs-12 bg-tertiary-400 border-radius-2 ellipsis fw-normal px-2"
-        style="outline:none;"
-        placeholder="Search"
-      />
-      <div class="d-flex align-items-center justify-content-center">
         <button
-          id="filter-btn"
-          class="filter-btn btn bg-backgroundDark d-flex align-items-center justify-content-center p-2 d-none
-          {showfilterDropdown ? 'filter-active' : ''}"
-          style="width: 32px; height:32px; position:relative"
-          on:click={() => (showfilterDropdown = !showfilterDropdown)}
+          style="height: 24px; width:24px;"
+          class="add-icon-container border-0 rounded-1 d-flex justify-content-center align-items-center {isHovered
+            ? 'collections-active'
+            : 'collections-inactive'}"
+          on:click|stopPropagation={showImportCollectionPopup}
```

Comment:
a check for `guest user `is required as guest user cant access import collection for now

Reply:
Resolved